client/src/main/java/net/spals/oembed4j/client/JerseyOEmbedClient.java

```diff
@@ -82,7 +73,7 @@ public void checkServerTrusted(final X509Certificate[] x509Certificates, final S
 
     public static JerseyOEmbedClient create(final OEmbedRegistry registry) {
         try {
-            // Configure the Jersey client
+            // 1. Configure the Jersey client
             // Setup SSL management
             // TODO: DO we need some real SSL management?
```

Comment:
Logged: https://github.com/spals/oembed4j/issues/7

FWIW, it was also on my notepad.